cloudflare_exporter/exporter.py

```diff
@@ -0,0 +1,35 @@
+import argparse
+
+from aiohttp import web
+from prometheus_client.core import REGISTRY
+
+from cloudflare_exporter.config import DEFAULT_HOST, DEFAULT_PORT
+from cloudflare_exporter.handlers import (handle_metrics,
+                                          handle_health)
+from cloudflare_exporter.collector import CloudflareCollector
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Cloudfalre prometheus exporter')
+    parser.add_argument('-t', '--token', type=str, required=True,
```

Comment:
Seems the "token" only auth is new. You can parhaps add email+token one

Reply:
ok that's something we can add in another review